### PR TITLE
Added docs for file helper

### PR DESCRIPTION
**Pull request by dfridrich**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes https://github.com/symfony/symfony/pull/18502 |
| Applies to | 3.1 |
| Fixed tickets |  |

### CODE REVIEW

**Comment by OskarStark** [comment]
LGTM 👍

**Comment by WouterJ** [comment]
@OskarStark it can't be as the feature has not yet been merged (and there still is some discussion on the implementation).

**Comment by OskarStark** [comment]
Oh sorry @WouterJ , i meant the current status looks good ;-)

File: book/controller.rst [hunk 1]
```diff
@@ -778,6 +778,43 @@ There are also special classes to make certain kinds of responses easier:
   :class:`Symfony\\Component\\HttpFoundation\\StreamedResponse`.
   See :ref:`streaming-response`.
 
+File helper
+~~~~~~~~~~~
+
+If you want to serve file use
```
Comment (xabbuh) [inline book/controller.rst]: [...] use the [...]

File: book/controller.rst [hunk 2]
```diff
@@ -778,6 +778,43 @@ There are also special classes to make certain kinds of responses easier:
...
+:method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::file`
+helper::
+
+    $this->file($file, $fileName = null, $disposition = ResponseHeaderBag::DISPOSITION_ATTACHMENT);
+
+You can pass:
+
+* Instance of
```
Comment (xabbuh) [inline book/controller.rst]: An instance of [...]

**Comment by WouterJ** [comment]
Hi @dfridrich.

I've rebased your PR and tweaked the wording a bit (so we can remove most of the paragraph and use the code examples more) in https://github.com/symfony/symfony-docs/pull/6732. I've kept your commit, so you'll still get the credits you deserve.

Thanks!

**Pull request closed without merging**